Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,25 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void LogFactory_SetsClassName()
+        {
+         // Arrange
+         LogFactory myLogFactory = new LogFactory();
+         myLogFactory.configureFileLogger("foobar");
+
+         FileLogger myLogger = (FileLogger)myLogFactory.CreateLogger(nameof(LogFactoryTests));
+                  
+         // Test
+         Assert.AreEqual(myLogger.className, nameof(LogFactoryTests));
+        }
+
+        [TestMethod]
+        public void LogFactory_ReturnsNull_IfFileLoggerNotConfigured()
+        {
+            FileLogger myLogger = (FileLogger)(new LogFactory()).CreateLogger("FileLogger");
```

Comment:
```suggestion
            LogFactory logFactory = new LogFactory();
```

We are testing the `LogFactory` class, so you should be creating a new `FileLogger` via the `LogFactory` class method `CreateLogger`.

Reply:
You would then need an extra line of code that creates a new BaseLogger - similar to what you did above on line 16.